### PR TITLE
Db Creation user service started auth

### PR DESCRIPTION
**Pull request by jjjkahn**

### CODE REVIEW

File: RiverLoggerApi.Repository/DataContext.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            await connection.ExecuteAsync(sql);
+        }
+
+        private async Task _initTables()
+        {
+            // create tables if they don't exist
+            using var connection = CreateConnection();
+            await _initUsers();
+
+            async Task _initUsers()
```
Comment (thibaultNicolas) [inline RiverLoggerApi.Repository/DataContext.cs:44]: You have two email

Comment (thibaultNicolas) [reply RiverLoggerApi.Repository/DataContext.cs:44]: Don't know if you should rename like EmailConfirmation

File: RiverLoggerApi.Repository/DbModels/UserDbo.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        public Guid UserId { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        public string LastName { get; set; }
+
+        [Required]
```
Comment (thibaultNicolas) [inline RiverLoggerApi.Repository/DbModels/UserDbo.cs:15]: I think you can add [EmailAddress] for validation but depends of the framework

File: RiverLoggerApi/Models/AuthenticateRequest.cs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 {
     public class AuthenticateRequest
     {
-        public required string Username { get; set; }
+        public required string Email { get; set; }
```
Comment (thibaultNicolas) [inline RiverLoggerApi/Models/AuthenticateRequest.cs:5]: Maybe we could use username and email auth ?

**Review by thibaultNicolas — CHANGES_REQUESTED** [review]

File: RiverLoggerApi.Repository/DbModels/UserDbo.cs (hunk 2 above)
Comment (jjjkahn) [reply RiverLoggerApi.Repository/DbModels/UserDbo.cs:15]: this should not be done in a dbo i will do it in the api level not at the repository level

File: RiverLoggerApi/Models/AuthenticateRequest.cs (hunk 3 above)
Comment (jjjkahn) [reply RiverLoggerApi/Models/AuthenticateRequest.cs:5]: like have them both?

Comment (jjjkahn) [reply RiverLoggerApi/Models/AuthenticateRequest.cs:5]: we will see when we come to that for now we will use only email

File: RiverLoggerApi.Repository/DataContext.cs (hunk 1 above)
Comment (jjjkahn) [reply RiverLoggerApi.Repository/DataContext.cs:44]: i will fix it good catch

**Review by thibaultNicolas — APPROVED** [review]

**Review by thibaultNicolas — APPROVED** [review]

**Pull request merged**